lib/matplotlib/style/core.py

```diff
@@ -84,6 +85,73 @@ def use(style):
                 raise IOError(msg % style)
 
 
+def get_substyles(styles):
```

Comment:
> Considering it, that would mean that in my example, parameters passed in the main stylesheet would have a higher priority than the ones in the a, b, c, would it not? Any stylesheet you call from an other one could have any parameter overridden? 

Yes, I think we're on the same page here. If `main` is the stylesheet you're calling `style.use` on and `a` is the stylesheet referenced in `main`, then the style parameters defined in `main` override the parameters in `a`.

> Can you confirm that I should remove the style feature from this PR?

Yes, that's what I had in mind. Since you're planning to break this out, it would be nice to break out some of the logic into smaller functions to make the overall flow a bit easier to follow.